spec.bs

```diff
@@ -841,9 +843,10 @@ An interest group ad is a [=struct=] with the following items:
 
 <dl dfn-for="interest group ad">
 : <dfn>render url</dfn>
-:: A [=URL=].
+:: A [=URL=]. If this ad wins the auction, this URL (or a URN that maps to this URL) will be returned by
+  `runAdAuction()`. This URL is intended to be loaded into an ad iframe (or a fenced frame).
 : <dfn>metadata</dfn>
-:: Null or a [=string=].
+:: Null or {{any}}. Extra arbitary information about this ad, passed to `generateBid()`.

```

Comment:
Additional JSON-serializable ad metadata, passed to `generateBid()`.

Reply:
I think it should be [=string=]. This structure is the internal data structure of an IG, converted from webIDL IG definition. Data stored in this data structure is already converted/serialized to a JSON (this `step`[https://github.com/WICG/turtledove/blob/main/spec.bs#L167] serialized an ad metadata to a JSON and this field is set to the JSON. This also applies to a few similar fields like `user bidding signals`[https://github.com/WICG/turtledove/blob/main/spec.bs#L821]

Reply:
Got it -- I had thought that the fact that we store strings is somewhat of an implementation decision, but it's somewhat implied by requiring that the object be JSON serializable.

I'll switch these back to string.